Enable broadcast of Facebook bot messages

B1tMaster opened the pull request:
**Description:**
it enables FB messenger broadcast ability to send messages to all people who interacted with the bot previously and have the chat open.

**Pull request in [home-assistant](https://github.com/home-assistant/home-assistant) (if applicable):** home-assistant/home-assistant#12459 

## Checklist:

- [ ] Branch: Fixes, changes and adjustments should be created against `current`. New documentation for platforms/components and features should go to `next`.
- [ ] The documentation follow the [standards][standards].

[standards]: https://home-assistant.io/developers/documentation/standards/

B1tMaster committed:
enable broadcast of FB bot messages

this should go together with PR https://github.com/home-assistant/home-assistant/pull/12459 
it enables FB messenger broadcast ability to send messages to all people who interacted with the bot previously and have the chat open.

c727 reviewed (changes requested):
FB -> Facebook

fix your indents -> 2x `_` for each level

B1tMaster committed:
fixes as per request from reviewer.

fixes as per request from reviewer. 
FB -> Facebook
fixed indents.  (not sure what 2x for each level was), but I fixed it as per sample above.

B1tMaster commented:
I have done changes as requested. please check. 
Can you also help me to figure out what is wrong with my code PR? There are no errors in my file, but it fails on code that i never touched or depend on.  https://github.com/home-assistant/home-assistant/pull/12459

Commit:
Update notify.facebook.markdown

fabaff commented on source/_components/notify.facebook.markdown:
```diff
@@ -117,3 +117,87 @@ script:
+      friendly_name: 'Wind'
+      icon_template: mdi:weather-windy
+      value_template: >-
+        {% if (states.sensor.pws_wind_kph.state | float + states.sensor.pws_wind_gust_kph.state | float)/2.0 > 18.0 -%}
```
All templates must be escaped otherwise they are rendered empty.

B1tMaster replied:
@fabaff I am very confused about this request. the documentation is very clear that ONLY single line templates must be escaped. these are NOT single line. And also, they work quite well on my home installation. So they do not render empty, I guarantee you. Please take a look at the documentation here: https://home-assistant.io/docs/automation/templating/

If you insist, I will put double quotes there, but it would be wrong in my view.

B1tMaster replied:
@fabaff I have done as you asked and escaped it.
The template is `multi-line` thus it should not be escaped. But since my PR is blocked, I did it anyway. Though I really believe it is not needed. Please check the documentation  here: https://home-assistant.io/docs/automation/templating/
It says:
`You must surround single-line templates with double quotes (") or single quotes (').`

B1tMaster replied:
@frenck i have done as asked, hope my original PR and this documentation can now go forward?

fabaff commented:
This PR was opened against the wrong the branch. New features need to go to `next`.

fabaff commented on source/_components/notify.facebook.markdown:
````diff
@@ -117,3 +117,87 @@ script:
                 payload: DEVELOPER_DEFINED_PAYLOAD_FOR_PICKING_BLUE
 ```
+
+You can now also use Facebook public beta broadcast API to push messages to ALL users who interacted with your chatbot on your page, without having to collect their number. This will scale to thousands of users. Facebook requires that this only be used for non-commercial purposes and they validate every message you send. Also note, your Facebook bot needs to be authorized for "page_subscritions"  ...
````
Typo: `page_subscritions`

B1tMaster replied:
I fixed and moved to correct branch.. Please review new version. How do I trigger new review for this?

frenck commented:
Needs to be rebased onto next. Marking it.

frenck committed:
:pencil2: Tiny fixes

frenck commented:
@B1tMaster Please take a look at the template escapes as requested by @fabaff. Thanks! 👍

B1tMaster committed:
Update notify.facebook.markdown

frenck commented:
Rebased the PR in order to fix the build issues.

@B1tMaster Reminder, could you please take a look at the remaining requests?

B1tMaster committed:
Update notify.facebook.markdown

B1tMaster reviewed (commented):
@frenck I removed the offending section completely. The contested example was not really needed for this documentation. Hope my PR (both code and documentation ) can now go in?

frenck committed:
:rocket: Trigger rebuild

frenck commented:
Pushed empty commit in order to trigger build again.
Awaiting build results.
